bundles/io/org.eclipse.smarthome.io.net/src/main/java/org/eclipse/smarthome/io/net/http/internal/ExtensibleTrustManagerImpl.java

```diff
@@ -118,6 +119,20 @@ public void checkServerTrusted(X509Certificate[] chain, String authType, SSLEngi
         }
     }
 
+    private X509ExtendedTrustManager getLinkedTrustMananger(X509Certificate[] chain, SSLEngine sslEngine) {
+        if (sslEngine != null) {
+            String peer = sslEngine.getPeerHost() + ":" + sslEngine.getPeerPort();
```

Comment:
What if `sslEngine.getPeerHost()` returns `NULL` 🙈

Reply:
I thought I checked the documentation this time ;-)

But indeed you are right, a `null` check is required here, thanks!

Reply:
Done